snomed/com.b2international.snowowl.snomed.core.rest/src/com/b2international/snowowl/snomed/core/rest/SnomedRf2ImportRestService.java

```diff
@@ -94,6 +83,10 @@
 			@RequestParam(name = "createVersions", defaultValue = "true")
 			final Boolean createVersions,
 			
+			@ApiParam(value = "Import should be allowed to progress when members of listed reference sets have missing referenced components", defaultValue = "false")
```

Comment:
Please remove the `defaultValue=false` declaration. That's not relevant when you are using a String type parameter.